### ISSUE TITLE
cannot build the docs

### ISSUE THREAD

**Issue by jrmolin** [cqctlang/l1#4]
i'm on ubuntu linux, and i can't build the documentation.

```
git clone https://github.com/cqctlang/l1
cd l1
make
make doc
```

![make-doc-error](https://user-images.githubusercontent.com/2355268/31678003-b847fb32-b33a-11e7-8a3b-25a65afaa943.png)

**Comment by flandr** [comment]
Hi, thanks for the report. Some of the examples (and the docs) refer to a now-out-of-date remote process control utility called sctl that is distributed separately from the l1 implementation (you can read about sctl briefly at http://cqctworld.org/). A snapshot is available but the current l1 runtime has diverged and they aren't presently compatible; you can't just grab the snapshot of sctl and expect it to work with this repo (though you could grab the snapshots from cqctworld.org of both l1 and sctl and they should work).

This may or may not be a blocker for you. If you're just trying to read the docs, you can do so without building as they are already up to date. If you're interested in using out-of-box Cinquecento as a remote process debugger, you're out of luck at the moment. Let me know what you're interested in, though, and perhaps I can help.

**Comment by jrmolin** [comment]
nothing's imminent. i just remember using it back when i worked with one of the maintainers. i liked the language, and i liked learning about how things were done and why. i have had more exposure to programming language design and implementation, so i wanted to pick this up again and play around with it. maybe look at embedding it in a larger system to see how that would compare to, say, lua (ugh). 

let me know if there's issues you don't have time to get to, and i'd be happy to poke around and collaborate.

tl;dr - it's not an issue. if the documentation is up-to-date, then closing this issue is totally fine.
